src/components/organisms/Modals/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,80 @@
+import { useAppDispatch } from "store/hooks";
+import { memo } from "react";
+import { Formik, Form } from "formik";
+
+import Button from "components/atoms/Button/Button";
+import Field from "components/molecules/Field/Field";
+
+import * as S from "./styles";
+import { setShowModal } from "features/app/appSlice";
+import { ChangePasswordValues } from "types/type";
+import { changePasswordSchema } from "utils/schemas";
+import { useChangePassword } from "lib/auth";
+
+const FormButton = memo(Button);
+
+const ChangePassword = () => {
+  const dispatch = useAppDispatch();
+  const initialValues: ChangePasswordValues = {
+    newPassword: "",
+    repeatNewPassword: "",
+    password: "",
+  };
```

Comment:
maybe move it to the useChangePassword, because it is used two times

Reply:
ChangePasswordValues are only used in the Modal for the Form?

Reply:
i mean you use initialValue in two places and you can extract them from this hook